.drone.yml

```diff
@@ -205,6 +229,28 @@ matrix:
     - PHP_VERSION: 5.6
       TEST_SUITE: owncloud-coding-standard
 
+    # phan
+    - PHP_VERSION: 7.2
```

Comment:
Question from me, the uninitiated.
Run ``phan`` on all PHP 7.* versions? Or just one of them? And which one?

Reply:
phan needs to run on the version we want to check against, as it requires php to convert it to its internals to check the AST.

phan thus should be run with every php version we support ( except 5.6 which does not have AST )

Reply:
pushed a change for this